modules/distr-tracing-tempo-troubleshoot-collecting-diagnostic-data-from-command-line.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="collecting-diagnostic-data-from-command-line_{context}"]
+= Collecting diagnostic data from the command line
+
+When submitting a support case, it is helpful to include diagnostic information about your cluster to Red Hat Support.
+You can use the `oc adm must-gather` tool to gather diagnostic data for resources of various types, such as `TempoStack` or `TempoMonolithic`, and the created resourced like `Deployment`, `Pod`, or `ConfigMap`.
+The `oc adm must-gather` tool creates a new pod that collects these data.
+
+.Procedure
+
+* From the directory where you want to save the collected data, run the `oc adm must-gather` command to collect the data:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=ghcr.io/grafana/tempo-operator/must-gather -- \
```

Comment:
I was going to say that these two modules are really repetitive and maybe to consider using the same module file but conditionals. I think there's enough differences that it's fine, but still something to consider.